src/shared/sam/localLambdaRunner.ts

```diff
@@ -154,6 +154,7 @@ export async function invokeLambdaFunction(
     if (!config.noDebug) {
         // Needed at least for dotnet case; harmless for others.
         samCliArgs.environmentVariables = {
+            ...samCliArgs.environmentVariables,
```

Comment:
Would it be possible to add test coverage to verify that debug config env vars are making their way to the cli call?